src/AbstractCollection.php

```diff
@@ -154,6 +154,9 @@ public function last()
         return $item;
     }
 
+    /**
+     * @inheritDoc
```

Comment:
When an extended method is missing `@inheritDoc`, any overrides for parameter/return types (in the docblock) will be lost.